lib/__tests__/harvest.test.ts

```diff
@@ -1,5 +1,5 @@
 import * as testServer from "../../server/server.js";
-import { getInstance } from "../../tests/utils";
+import { getInstanceForBrowser as getInstance } from "../../tests/utils";
```

Comment:
Since the function is exactly the same as the `browser` module, why not using the module?

Reply:
It's almost the same. `browser` module exports an instance, and `getInstanceForBrowser` is `a function returning an instance`.
So I've made a `_instance.ts` file to have `getInstanceForBrowser` function and `browser` module now uses this.
The same applies to `node` module.

6f23ec1